CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Acra ChangeLog
 
+## [0.92.0](https://github.com/cossacklabs/acra/releases/tag/0.92.0), February 17th 2021
+
```

Comment:
TLDR: this release brings stability and performance fixes to AcraServer and AcraTranslator. It officially deprecates usage of AcraConnector in favour of TLS everywhere. Some default configuration params are changed in favour of more secure & better performance settings.